### ISSUE TITLE
Mullvad not working

### ISSUE THREAD

**Issue by loscamos** [haugene/vpn-configs-contrib#243]
### Is there a pinned issue for this?

- [X] I have read the pinned issues

### Is there an existing or similar issue for this?

- [X] I have searched the existing issues

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to the container/transmission?

- [X] I have checked the container repo for issues

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Config used

version: "3.3"
services:
  transmission-openvpn:
    cap_add:
      - NET_ADMIN
    volumes:
      - /srv/dev-disk-by-uuid-eb879951-c7c5-42c8-9543-2924b917b18e/Docker/Transmission_New:/data
      - /srv/dev-disk-by-uuid-eb879951-c7c5-42c8-9543-2924b917b18e/Docker/Transmission_New:/config
    dns:
      - 1.1.1.1
    devices:
      - /dev/net/tun
    sysctls:
      - net.ipv6.conf.all.disable_ipv6=0
    environment:
      - OPENVPN_PROVIDER=MULLVAD
      - OPENVPN_CONFIG=it_all
      - OPENVPN_USERNAME=********************
      - OPENVPN_PASSWORD=m
      - LOCAL_NETWORK=192.168.188.0/24
      - PUID=1000
      - PGID=1000
    ports:
      - 9091:9091
      - 8888:8888
    restart: always
    image: haugene/transmission-openvpn:latest

### Current Behavior

I am not able to connect with Mullvad. Tried other providers and everything works. 

### Expected Behavior

Mullvad to connect

### How have you tried to solve the problem?

Updated OMV to latest version
Updated Docker
Removed the container and all dependencies
Tried different docker configurations

### Log output

Starting container with revision: 81277fb6f8ae6aeb57f913172d4d45c7af8c0b31

TRANSMISSION_HOME is currently set to: /config/transmission-home

Creating TUN device /dev/net/tun

Using OpenVPN provider: MULLVAD

Running with VPN_CONFIG_SOURCE auto

No bundled config script found for MULLVAD. Defaulting to external config

Will get configs from https://github.com/haugene/vpn-configs-contrib.git

Repository is already cloned, checking for update

Already up to date.

Already on 'main'

Your branch is up to date with 'origin/main'.

Found configs for MULLVAD in /config/vpn-configs-contrib/openvpn/mullvad, will replace current content in /etc/openvpn/mullvad

Starting OpenVPN using config it_all.ovpn

Modifying /etc/openvpn/mullvad/it_all.ovpn for best behaviour in this container

Modification: Point auth-user-pass option to the username/password file

Modification: Change ca certificate path

Modification: Change ping options

Modification: Update/set resolv-retry to 15 seconds

Modification: Change tls-crypt keyfile path

Modification: Set output verbosity to 3

Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop

Modification: Updating status for config failure detection

Setting OpenVPN credentials...

adding route to local network 192.168.188.0/24 via 172.26.0.1 dev eth0

2023-10-09 06:09:48 Note: option tun-ipv6 is ignored because modern operating systems do not need special IPv6 tun handling anymore.

2023-10-09 06:09:48 OpenVPN 2.5.5 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 14 2022

2023-10-09 06:09:48 library versions: OpenSSL 3.0.2 15 Mar 2022, LZO 2.10

2023-10-09 06:09:48 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts

2023-10-09 06:09:48 TCP/UDP: Preserving recently used remote address: [AF_INET]89.40.182.210:1194

2023-10-09 06:09:48 Socket Buffers: R=[212992->425984] S=[212992->425984]

2023-10-09 06:09:48 UDP link local: (not bound)

2023-10-09 06:09:48 UDP link remote: [AF_INET]89.40.182.210:1194

### Environment

```markdown
- OS: OMV
- Docker: 24.0.6, build ed223bc
```

**Comment by pkishino** [comment]
no error can be seen in the logs.. not sure you posted it all..
it_all would be the correct name as per above..
did you check the mullvad folder to see what the other configs are named?

**Comment by loscamos** [comment]
It's unbelievable. I spent 6 hours finding the issue. I wouldn't notice it if it wasn't for your comment... I just checked the mullvad folder created after deploying the container. The container starts with all the other ovpn, besides it_all. I opened some of the others ovpn file (ae_all, al_all) and checked them. I then opened the it_all and compared the server to the one present on the mullvad website. There is an error just in the it_all ovpn file generated.

These is the servers listed in the generated file (the wrong one):

remote it-mil-ovpn-101. relays. mullvad.net 1194
remote it-mil-ovpn-102. relays.mullvad. net 1194
remote it-mil-ovpn-103. relays.mullvad. net 1194 
remote it-mil-ovpn-104. relays.mullvad. net 1194,

If you check on mullvad website these are correct:

remote it-mil-ovpn-201. relays. mullvad.net 1194
remote it-mil-ovpn-202. relays. mullvad.net 1194

The others are no longer working.
